.github/workflows/installer-build-check.yml

```diff
@@ -0,0 +1,79 @@
+name: Build Test Installers
+
+# Job will run nightly at 03:05 EDT / 02:05 CDT
+# Time below is UTC
+on:
+  schedule:
+    - cron: "5 7 * * *"
+  workflow_dispatch:
+  push:
+    branches:
+      - gha-scheduled-*
+
+jobs:
+  build-installers:
+    name: Build Installers
+    runs-on: macos-latest #needs macos for apple notarization
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Download liquibase-artifacts
+        uses: liquibase/action-download-artifact@v2-liquibase
+        with:
+          workflow: build.yml
+          workflow_conclusion: success
+          branch: master
+          name: liquibase-artifacts
+          path: re-version/final
+
+      - name: Set up JDK for GPG
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
```

Comment:
@ap-liquibase @nvoxland should this be java 8 or java 11?

Reply:
also should we add the `cache` options that the setup-java action has?

Reply:
I think for builds were using java 8